### PR TITLE
Update task pages

### PR DESCRIPTION
**Pull request by merveenoyan**
- Update feature extraction with mteb, use cases, models
- Add new zero shot object detection and depth estimation models and apps
- Update image-to-text ones with new models
- Add new object detection model, app and dataset

### CODE REVIEW

File: packages/tasks/src/tasks/depth-estimation/data.ts [hunk 1]
```diff
@@ -24,8 +28,8 @@ const taskData: TaskDataCustom = {
 	metrics: [],
 	models: [
 		{
-			description: "Strong Depth Estimation model trained on 1.4 million images.",
-			id: "Intel/dpt-large",
+			description: "Cutting-edge depth estimation model.",
```
Comment (Vaibhavs10) [inline packages/tasks/src/tasks/depth-estimation/data.ts:31]: QQ: Does adding the Depth Anything v2 optimised for Coreml here as a point of reference and as a kickback to apple org makes sense?

Could also be DETR too

cc: @osanseviero too

Comment (merveenoyan) [reply packages/tasks/src/tasks/depth-estimation/data.ts:31]: feel free to make a commit suggestion on a separate row, and I find it relevant so would be nice addition

File: packages/tasks/src/tasks/feature-extraction/about.md [hunk 2]
```diff
@@ -1,9 +1,21 @@
...
 
+### Retrieval and Reranking
+
+Retrieval is the process of retrieving relevant documents or information based on a user's search query. In the context of NLP, retrieval systems aim to find relevant text passages or documents from a large corpus of data that match the user's query. The goal is to return a set of results that are likely to be useful to the user. On the other hand, reranking is a technique used to improve the quality of retrieval results by reordering them based on their relevance to the query.
+
+### Retrieval Augmented Generation
+
+Retrieval Augmented Generation (RAG) is a technique where the user inputs to generative models first queried through a knowledge base, and the most relevant information from the knowledge base are used to augment the prompt to reduce hallucinations during generation. Feature extraction models (primarily, retrieval and reranking models) can be used in RAG to reduce model hallucinations and ground the model.
```
Comment (osanseviero) [inline packages/tasks/src/tasks/feature-extraction/about.md:13]:
```suggestion
Retrieval-augmented generation (RAG) is a technique in which user inputs to generative models are first queried through a knowledge base, and the most relevant information from the knowledge base is used to augment the prompt to reduce hallucinations during generation. Feature extraction models (primarily retrieval and reranking models) can be used in RAG to reduce model hallucinations and ground the model.
```

File: packages/tasks/src/tasks/feature-extraction/about.md [hunk 3]
```diff
@@ -1,9 +1,21 @@
...
+
 ## Inference
 
+You can infer feature extraction models using `pipeline` of transformers library.
```
Comment (osanseviero) [inline packages/tasks/src/tasks/feature-extraction/about.md:17]: Would also be cool to add a `sentence-transformers` example @tomaarsen

File: packages/tasks/src/tasks/object-detection/data.ts [hunk 4]
```diff
@@ -4,9 +4,14 @@ const taskData: TaskDataCustom = {
 	datasets: [
 		{
 			// TODO write proper description
-			description: "Widely used benchmark dataset for multiple Vision tasks.",
+			description: "Widely used benchmark dataset for multiple vision tasks.",
 			id: "merve/coco2017",
 		},
+		{
+			// TODO write proper description
+			description: "Multi-task computer vision benchmark.",
```
Comment (osanseviero) [inline packages/tasks/src/tasks/object-detection/data.ts:12]: TODO

**Comment by merveenoyan** [comment]
@osanseviero can you approve if ready?

**Review by osanseviero — APPROVED** [review]
LGTM!

**Pull request merged**